src/greenlight/test.clj

```diff
@@ -47,20 +47,38 @@
              :kind? vector?
              :min-count 1))
 
+(defn- contains-ns? [coll ns]
+  (not (empty? (reduce-kv (fn [m k v]
+                            (if (= ns (namespace k))
+                              (assoc m k v)
+                              m)) {} coll))))
+
+(defn- attr-map? [o]
```

Comment:
Can you put the argument vector on the next line to conform to existing styling?

Reply:
Aye, I will make the change